src/SearchResults.js

```diff
@@ -1,9 +1,22 @@
-import React from "react";
+import React, { useState } from "react";
 import moment from "moment";
 
 const SearchResults = props => {
+  const [rowSelect, setRowSelect] = useState("");
+
+  const changeBackground = data => {
+    if (data === rowSelect) {
+      setRowSelect();
+    } else {
+      setRowSelect(data);
```

Comment:
This is a really good attempt, but it looks like if you click on any row, then all of them become highlighted. The exercise asks you to make each row *independent*, meaning that multiple rows can be highlighted at once.

We'll look at a simple solution on Sunday, but the idea is to think carefully about how we could split up our components so that they can have their own state.